components/scream/src/share/field/field.hpp

```diff
@@ -10,7 +10,6 @@
 #include "ekat/kokkos/ekat_kokkos_utils.hpp"
 #include "ekat/kokkos/ekat_subview_utils.hpp"
 
-#include <Kokkos_MemoryTraits.hpp>
```

Comment:
I don't fully understand why kokkos discourages including individual header (and just use `Kokkos_Core.hpp`). I usually try to include what's necessary, since it helps me understand what the current file does/needs. That said, it's no big deal, so I don't really care. :)